docs/RUNNING.md

```diff
@@ -2,31 +2,39 @@
 
 ## Table of Contents
 
-- [Running components automatically](#running-components-automatically)
-- [Environment Variables](#environment-variables)
-- [Troubleshooting](#troubleshooting)
+- [Running](#running)
```

Comment:
I don't think the table of contents is worthwhile given how short it is.

Reply:
This has been resolved now.

Reply:
It is still here. Am I missing something?

Reply:
Sorry, it was not removed. Check again now.